ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-master-presubmits.yaml

```diff
@@ -47,7 +47,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )build,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
This usually causes tests to have to be ran manually. are you sure the behavior is different when skip_if_only_changed is set?

Reply:
That's a good question. [The jobs should be set to run conditionally based on the contents of the pull request](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes).